src/Tribe/Views/V2/Assets.php

```diff
@@ -471,6 +471,22 @@ public function register() {
 				],
 			]
 		);
+
+		tribe_asset(
+			$plugin,
+			'tribe-admin-v2-single-blocks',
+			'tribe-admin-single-blocks.css',
+			[ 
+				'tec-variables-full',
+				'tec-variables-skeleton',
```

Comment:
https://github.com/the-events-calendar/tribe-common/pull/1716 should make this unnecessary